tests/fixtures/group.json

```diff
@@ -667,6 +667,43 @@
 		},
 		"status_code": 200
 	},
+  "delete_external_feed": {
+		"method": "DELETE",
+		"endpoint": "groups/1/external_feeds/1",
+		"data": {
+			"id": 1,
+			"display_name": "My Blog",
+			"url": "http://example.com/myblog.rss"
+		},
+		"status_code": 200
+	},
+	"list_external_feeds": {
+		"method": "GET",
+		"endpoint": "groups/1/external_feeds",
+		"data": [
+			{
+				"id": 1,
+				"display_name": "My Blog",
+				"url": "http://example.com/myblog.rss"
+			},
+			{
+				"id": 2,
+				"display_name": "My Blog 2",
+				"url": "http://example.com/myblog2.rss"
+			}
+		],
+        "status_code": 200
+	},
+	"create_external_feed": {
+		"method": "POST",
+		"endpoint": "groups/1/external_feeds",
+		"data": {
+			"id": 1,
+			"display_name": "My Blog",
+			"url": "http://example.com/myblog.rss"
+		},
+		"status_code": 200
+	}
```

Comment:
I made a mistake when merging in `develop`. Please add a comma after this bracket.